packages/@aws-cdk/aws-servicecatalog/lib/product-stack-asset-bucket.ts

```diff
@@ -0,0 +1,114 @@
+import { IBucket, BlockPublicAccess, Bucket, BucketEncryption } from '@aws-cdk/aws-s3';
+import { BucketDeployment, ISource, Source } from '@aws-cdk/aws-s3-deployment';
+import * as cdk from '@aws-cdk/core';
+import { Construct, IConstruct } from 'constructs';
+import { hashValues } from './private/util';
+/**
+ * Product stack asset bucket props.
+ */
+export interface ProductStackAssetBucketProps {
+  /**
+   * Name of s3 asset bucket deployed
+   *
+   * @default - None
```

Comment:
The default is not none... a bucket name is generated if this property is not supplied.